packages/graphback-datasync/tests/DataSyncMongoDBTests.ts

```diff
@@ -173,6 +173,23 @@ describe('Delta Queries', () => {
   let context: Context;
   afterEach(async () => context.server.stop());
 
+  it('creates index for the lastUpdateAt field', async () => {
+
+    context = await createTestingContext(postSchema);
+
+    const indexes = await context.db.collection('post').indexes();
+
+    expect(indexes[1]).toMatchObject(
+      {
+        "key": {
+          "_lastUpdatedAt": 1
```

Comment:
```suggestion
          [DataSyncFieldNames.lastUpdatedAt]: 1
```
Maybe this would be better

Reply:
Thanks, applied.